### DISCUSSION TITLE
metamask erc20 to phantom ( Sol ) error

### DISCUSSION THREAD

**Discussion by slickrider** [wormhole-foundation/wormhole#496] [Problems / bugs] [answered]
i sent my eth from my metamask to my my phantom wallet ( sol ) and cant redeem my tokens , coins where taken from metamask but not delivered to my phantom wallet

<img width="1680" alt="Screen Shot 2021-09-27 at 2 46 40 PM" src="https://user-images.githubusercontent.com/91293229/134968130-07948f0d-1b36-49d1-9926-8a431f4f7853.png">

**Comment by leoluk** [comment, marked as answer]
Please provide a transaction ID

**Comment by slickrider** [comment]
The tokens have been sent!
0x7ea21cf1491ac8b635082aa2198d17a4e0ac28dbcfc4800f3032e2dbaaac4373

**Reply by leoluk** [reply]
Looks like the tokens were successfully sent to Solana, then sent back to Eth: https://explorer.solana.com/address/DYzAxhQbVtmYcjgtncQfLYbSHporqaT12AHDzQBnRffc

**Comment by slickrider** [comment]
They are in my phantom wallet as an uknown coin


Sent from Yahoo Mail for iPhone


On Tuesday, October 19, 2021, 2:10 PM, Leopold Schabel ***@***.***> wrote:




Looks like the tokens were successfully sent to Solana, then sent back to Eth: https://explorer.solana.com/address/DYzAxhQbVtmYcjgtncQfLYbSHporqaT12AHDzQBnRffc

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
